README.md

```diff
@@ -1,8 +1,8 @@
 ## Usage
 
-Create a script called `ubuntu` that looks like:
+Create a script called `bash` that looks like:
```

Comment:
I don't know why I chose `ubuntu` as the script name to start with. That was a poor choice.